### PR TITLE
Merge pull request #1297 from opengisch/sparkles

### PR DESCRIPTION
**Pull request by domi4484**
Backports Sparkles to 1.7

### CODE REVIEW

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1298-39a4c0417d3483a06408ae55fe348332922a13db-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1298-39a4c0417d3483a06408ae55fe348332922a13db-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1298-39a4c0417d3483a06408ae55fe348332922a13db-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1298-39a4c0417d3483a06408ae55fe348332922a13db-x86_64.apk)

File: src/qml/WelcomeScreen.qml [hunk 1]
```diff
@@ -266,6 +267,108 @@ Page {
...
+
+    ParticleSystem {
+      id: unicorns
+      running: imageDialLogo.value < 0.1
+    }
+
+    ImageParticle {
+      anchors.fill: parent
+      system: particles
+      source: "qrc:///particleresources/star.png"
```
Comment (m-kuhn) [inline src/qml/WelcomeScreen.qml:312]: @domi4484 should this file be present in this pr?

Comment (domi4484) [reply src/qml/WelcomeScreen.qml:312]: No, this is provided by Qt sinc V5.2.
https://doc.qt.io/qt-5/qml-qtquick-particles-imageparticle.html

**Review by m-kuhn — APPROVED** [review]
LGTM :milk_glass:

**Comment by signedav** [comment]
Thanks @domi4484

**Pull request merged**